test-utils/runtime/src/lib.rs

```diff
@@ -180,6 +180,17 @@ impl ExtrinsicT for Extrinsic {
 	}
 }
 
+impl Dispatchable for Extrinsic {
+	type Origin = Origin;
+	type Trait = ();
+
+	fn dispatch(self, _: Origin) -> sp_runtime::DispatchResult {
+		match self {
+			_ => panic!("the extrinsic is not supposed to be dispatched"),
+		}
```

Comment:
```suggestion
		panic!("the extrinsic is not supposed to be dispatched"),
```

If we want to panic then might as well do it brief :D

Reply:
This code was removed from the PR.